_posts/0000-01-02-cankiriscioglu.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![you-did-it-24a783eb67](https://user-images.githubusercontent.com/64472964/80488541-2befe400-8967-11ea-9a15-f4888c77aee5.jpg)